src/mon/MonCommands.h

```diff
@@ -527,9 +527,10 @@ COMMAND("osd crush add " \
 	"add or update crushmap position and weight for <name> with <weight> and location <args>", \
 	"osd", "rw", "cli,rest")
 COMMAND("osd crush set-device-class " \
-	"name=id,type=CephOsdName " \
-	"name=class,type=CephString ", \
-	"set the <class> of the device <name>", \
+        "name=class,type=CephString " \
+	"name=ids,type=CephString,n=N", \
+	"set the <class> of the osd(s) <id> [<id>...]," \
+        "or use <all|any|*> to set all.", \
```

Comment:
This breaks:

<pre>
2017-07-12T22:14:25.296 INFO:tasks.workunit.client.0.smithi172.stderr:+ ceph osd crush set-device-class osd.0 ssd
2017-07-12T22:14:25.447 INFO:tasks.workunit.client.0.smithi172.stderr:Error EINVAL: Expected option value to be integer, got 'ssd', unable to parse osd id:"ssd".
</pre>
/a/sage-2017-07-12_19:30:01-rados-wip-sage-testing-distro-basic-smithi/1392480

..and other tests in this run.  let's drop this commit?